packages/app/src/pages/Settings.vue

```diff
@@ -17,4 +17,21 @@ query Settings {
 
 const query = useQuery({ query: SettingsDocument })
 
+gql`
+subscription Settings_cloudProjectUpdate {
```

Comment:
To use this feature, is it required that the field be a subscription? Whatever wiring is required, I think we should add as much documentation as we can for what these new directives do and when/why you'd implement them